docs/storybook/src/deprecated/Icon.stories.tsx

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 import React from "react";
 import type { Decorator, Meta, StoryObj } from "@storybook/react";
-import placeholderSvg from "@bentley/icons-generic/icons/placeholder.svg";
+import placeholderSvg from "../../assets/placeholder.svg";
```

Comment:
iTwinUI-Icons has a [placeholder icon](https://itwin.github.io/iTwinUI-icons/#icons-placeholder), can that be used?

Reply:
In this case, we want to use the SVG directly to test that the component still supports that.